### PR TITLE
Weights and Reps Page

### PR DESCRIPTION
**Pull request by Karumu05**
This pull request will add the Weights and Rep page with the functionality to "addExcerise". The actual new exercise will not be added yet as waiting on the backend to be completed.

### CODE REVIEW

**Review by Siffster85 — APPROVED** [review]
I can't see anything obvious, but let's wait for Elena :)

File: app/(tabs)/plan/resistancePage.tsx [hunk 1]
```diff
@@ -1,7 +1,197 @@
...
+    setReps('');
+  };
+
+  const handleSubmit = async () => {
+    try {
+      //Add data to the API
+    } catch {
+      alert('Something went wrong please try again');
+    }
+    router.replace('/(tabs)/plan/workout');
```
Comment (elenatrr) [inline app/(tabs)/plan/resistancePage.tsx]: finally {router.replace('/(tabs)/plan/workout');}   ??

File: app/(tabs)/plan/resistancePage.tsx [hunk 2]
```diff
@@ -1,7 +1,197 @@
...
+  };
+
+  const handleRemoveSet = () => {
+    if (data.length > 0) {
+      setData(prevData => prevData.slice(0, prevData.length - 1));
+    }
+  };
+
+  const Item = ({ exerciseName, weight, reps }: Excersie) => {
```
Comment (elenatrr) [inline app/(tabs)/plan/resistancePage.tsx]: you get the error about categories because you set **const Item** type to **Excersie**

**Review by elenatrr — CHANGES_REQUESTED** [review]

File: app/(tabs)/plan/resistancePage.tsx (hunk 1 above)
Comment (Karumu05) [reply app/(tabs)/plan/resistancePage.tsx]: This will re route back to page when the data has been added to the API

**Review by elenatrr — APPROVED** [review]

**Pull request merged**